catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/controller/SpriteController.java

```diff
@@ -151,6 +163,13 @@ public void delete(Sprite spriteToDelete) {
 				Log.e(TAG, Log.getStackTraceString(e));
 			}
 		}
+		File projectDirectory = ProjectManager.getInstance().getCurrentProject().getDirectory();
```

Comment:
this just doesnt break anything right now because backpack is not handling user variables correctly. (so this is working because of another bug)

since this delete method is also used by the backpack when one deletes a sprite in backpack, it tries to delete the same thing in the currently loaded project aswell.

Reply:
Ok. A cleanup routine every project load instead would also solve this problem, wouldn't it?